prbench/src/prbench/envs/tidybot/objects.py

```diff
@@ -268,3 +269,299 @@ def __repr__(self) -> str:
             f"Cube(name='{self.name}', joint_name='{self.joint_name}', "
             f"size={self.size}, rgba='{self.rgba}', mass={self.mass})"
         )
+
+
+class MujocoFixture:
+    """Base class for MuJoCo fixtures (static objects)."""
+
+    def __init__(
+        self,
+        name: str,
+        env: Optional[MujocoEnv] = None,
+    ) -> None:
+        """Initialize a MujocoFixture.
+
+        Args:
+            name: Name of the fixture body in the XML
+            env: Reference to the environment
+        """
+        self.name = name
+        self.env = env
+
+    def sample_pose_in_region(
```

Comment:
I'm not sure if this makes sense as a general method for fixtures, if fixtures are really just static objects. the docstring mentions tables, which would make sense. but what about a fixture like a lamp or something?